laravel/app/Console/Commands/StatsDownload.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Services\Stats\DownloadService\IDownloadService;
+use App\Services\Stats\DownloadService\Result;
+use App\Services\LoggingService\ILoggingService;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Lang;
+
+/**
+ * Class StatsDownload
+ *
+ * To execute this command, from the SSH shell execute:
+ *
+ *      cd /vagrant/laravel
+ *      php artisan stats:download
+ *
+ * @package App\Console\Commands
+ */
+class StatsDownload extends Command
```

Comment:
This it the stats download task. This can be executed from the commandline via:

```
vagrant ssh
cd /vagrant/laravel
php artisan stats:download
```

Or executed via task scheduler, or PHP code from another location